packages/manager/apps/key-management-service/src/components/credential/create/identities/tile/IdentitiesUserTile.component.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { Identity } from '@/types/identity.type';
+import IdentitiesTileText from './IdentitiesTileText.component';
+import IdentitiesBaseTile from './IdentitiesBaseTile.component';
+import IdentitiesStatusBadge from '../badge/IdentitiesStatusBadge.component';
+
+type IdentitiesUserTileProps = {
+  user: Identity;
+  selectedUserList: Identity[];
+  setSelectedUserList: Dispatch<SetStateAction<Identity[]>>;
+};
+
+const IdentitiesUserTile = ({
+  user,
+  selectedUserList,
+  setSelectedUserList,
+}: IdentitiesUserTileProps) => {
+  const { t } = useTranslation('key-management-service/credential');
+  const [isChecked, setIsChecked] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (selectedUserList.find((userInList) => userInList.urn === user.urn)) {
+      setIsChecked(true);
+    }
+  }, [selectedUserList, user]);
+
+  const updateUserInList = (isInList: boolean) => {
+    if (isInList) {
+      setSelectedUserList((prevUserList) => [...prevUserList, user]);
+    } else {
+      setSelectedUserList((prevUserList) =>
+        prevUserList.filter((userInList) => userInList.urn !== user.urn),
+      );
+    }
+  };
+
+  return (
+    <IdentitiesBaseTile
+      title={user.login}
+      urn={user.urn}
+      updateCallback={updateUserInList}
+      isChecked={isChecked}
+      setIsChecked={setIsChecked}
+    >
+      <span>
+        <OsdsText>
+          {t(
+            'key_management_service_credential_create_identities_user_tile_email_label',
+          )}
+          :
+        </OsdsText>
+        <IdentitiesTileText>{user.email}</IdentitiesTileText>
+      </span>
+      <span>
+        <OsdsText>
+          {t(
+            'key_management_service_credential_create_identities_user_tile_group_label',
+          )}
+          :
+        </OsdsText>
+        <IdentitiesTileText>{user.group} </IdentitiesTileText>
+      </span>
+      <span>
+        <OsdsText>
+          {t(
+            'key_management_service_credential_create_identities_user_tile_identity_label',
+          )}
+          :
+        </OsdsText>
+        <IdentitiesTileText>{user.urn}</IdentitiesTileText>
+      </span>
+      <span>
+        <IdentitiesStatusBadge status={user.status} inline />
```

Comment:
```suggestion
        <IdentitiesStatusBadge
          status={user.status}
          inline
          size={ODS_CHIP_SIZE.sm}
        />
```
A smaller Chip will be better in that case